index.html

```diff
@@ -1090,27 +1073,18 @@ <h4>Proof Configuration (eddsa-jcs-2022)</h4>
 Let <var>proofConfig</var> be a clone of the <var>options</var> object.
             </li>
             <li>
-If <var>options</var>.<var>type</var> is not set to `DataIntegrityProof` and
+If <var>proofConfig</var>.<var>type</var> is not set to `DataIntegrityProof` and
```

Comment:
Should this be `and/or` instead of `and`?
```suggestion
If <var>proofConfig</var>.<var>type</var> is not set to `DataIntegrityProof` and/or
```